src/resources/elements/typefield.js

```diff
@@ -40,10 +40,10 @@ export class Typefield extends Field {
    */
   keyPlaceholder = '';
   /**
-   * Whether or not to show the chosen type in the value of this field.
-   * @type {Boolean}
+   * The key where to put the type of this field.
+   * @type {String}
    */
-  showType = true;
+  typeKey = 'x-oad-type';
```

Comment:
What is 'x-oad-type'? Descriptive comment here would help. Imagine you are explaining each line of the code to a newcomer.